routes/owner.js

```diff
@@ -0,0 +1,84 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const { Hotel, validate } = require('../models/hotel')
+const { Reservation } = require('../models/reservation')
+const router = express.Router()
+
+// TODO: add auth middlewear, when it will be ready
+
+router.get('/hotels', async (req, res) => {
+  const hotel = await Hotel.find()
+
+  res.status(200).send(hotel)
+})
+
+router.post('/hotel', async (req, res) => {
+  const { error } = validate(req.body)
+  if (error) return res.status(400).send(error.details[0].message)
+
+  const hotel = new Hotel(req.body)
+
+  await hotel.save()
+
+  res.status(200).send(hotel)
+})
+
+router.put('/hotel/:id', async (req, res) => {
+  const { error } = validate(req.body)
+  if (error) return res.status(400).send(error.details[0].message)
+
+  const hotel = await Hotel.findByIdAndUpdate(req.params.id, req.body)
+
+  if (!hotel) {
```

Comment:
honestly i dont understand what happened :p We find hotel by id and update it.  then we check if hotel exists XDDDD
you can wrap it in try catch block :p